### PR TITLE
DOCS-12430 SAP Successfactors 4.1

### PR DESCRIPTION
**Pull request by sathya0**

### CODE REVIEW

File: sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-reference.adoc [hunk 1]
```diff
@@ -324,6 +324,11 @@ You might receive duplicate records when using the *Query* operation with pagina
 | Page Size a| Number |  |  100 |
 | Top a| Number |  |  |
 | Skip a| Number |  |  |
+| Pagination Type a| Enumeration, one of:
+
+** CLIENT
+** CURSOR
```
Comment (GeorgeTrifSV) [inline sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-reference.adoc:330]: Can you leave a note that:

> Snapshot Pagination does not support `$top` and `$skip`

> Cursor Pagination does not support `$orderby`

**Review by GeorgeTrifSV — CHANGES_REQUESTED** [review]

**Review by GeorgeTrifSV — DISMISSED** [review]

File: sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-upgrade-migrate.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|3.2.x |4.0.0
+|===
+
+== Changes in This Release
+
```
Comment (dhirsch1) [inline sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-upgrade-migrate.adoc:14]: Do you want to include an update for v4.1?

File: sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-xml-maven.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
+= SAP SuccessFactors Connector XML and Maven Support - Mule 4
```
Comment (dhirsch1) [inline sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-xml-maven.adoc]:
```suggestion
= SAP SuccessFactors Connector 4.1 XML and Maven Support - Mule 4
```

File: sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-upgrade-migrate.adoc [hunk 4]
```diff
@@ -0,0 +1,77 @@
+= Upgrade SAP SuccessFactors Connector to Version 4.0 - Mule 4
```
Comment (dhirsch1) [inline sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-upgrade-migrate.adoc:1]: I want to discuss how to deal with update topics like this one at a connector meeting. We should probably change 4.0 to 4.x and add the new functionality for the minor release but don't do anything now.

File: sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-config-topics.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
+= SAP SuccessFactors - Additional Configuration Information - Mule 4
```
Comment (dhirsch1) [inline sap-successfactors/4.1/modules/ROOT/pages/sap-successfactors-connector-config-topics.adoc]:
```suggestion
= SAP SuccessFactors 4.1 - Additional Configuration Information - Mule 4
```

**Review by dhirsch1 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**